### DISCUSSION TITLE
Error when installing .deb package libcurl4 is not installable.

### DISCUSSION THREAD

**Discussion by clt16** [OpenIDC/mod_auth_openidc#531] [General]
When attempting to install the following:

libapache2-mod-auth-openidc_2.4.5-1.bionic+1_amd64.deb 

I am seeing the following error:

The following packages have unmet dependencies:
 libapache2-mod-auth-openidc : Depends: libcurl4 (>= 7.16.2) but it is not installable

I have already installed the following package prior to installing openidc, 
libcjose0_0.6.1.5-1.bionic+1_amd64.deb

This is on Debian 9 Linux.

**Comment by zandbelt** [comment]
installing Ubuntu Bionic package on a Debian system is not possible; the latest Debian packages are available under the commercial support program, you can build binaries yourself, or use older versions from the standard package repositories
